src/main/java/seedu/address/logic/parser/CalledCommandParser.java

```diff
@@ -13,21 +16,27 @@
  */
 public class CalledCommandParser implements Parser<CalledCommand> {
 
+    public static final String MESSAGE_INDEX_NOT_PARSED = "The index entered is not within the allowed range. "
+            + "The index for display cannot be bigger than 2147483647 (i.e., MAX_VALUE).";
```

Comment:
Since quite a few classes use this, do you think its better to throw it into the Messages class? So that you don't have to create the exact same string multiple times.
(Located at: Commons -> Core -> Messages)

Reply:
okay good point, i'll do that